content/talks/_index.html

```diff
@@ -0,0 +1,23 @@
+---
+title: "Talks"
+date: 2020-10-20T22:26:10+11:00
+draft: false
+---
+<section class="section">
+    <div class="container is-size-5">
+        <div class="columns is-centered">
+            <div class="column is-12">
```

Comment:
It's a bit wide on desktop. Adding "is-8-desktop" would make it the same width as the showcase on desktop and above.

Reply:
Done ✅